### PR TITLE
feat: enable the percentage limit feature

### PR DESCRIPTION
**Pull request by felix021**
#### What type of PR is this?

feat

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.
- [ ] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en: Kitex v0.7.2 provides a new feature for precisely limiting the percentage of retry requests, which should be more practical than the old way (depending on the error rate), especially for backup request. Optimize the document by the way.
zh(optional):

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,10 +143,12 @@ Provide the mechanism to custom the nacos parameter `vo.ConfigParam`.
...
 Example:
-```
-configDataID: ServiceName.limit
 
+> configDataID: ServiceName.limit
+
+```json
 {
   "connection_limit": 100,
   "qps_limit": 2000
```
Comment (whalecold) [inline README.md:154]: 这里的配置要更新下吗

Comment (felix021) [reply README.md:154]: 不需要，retry 的初始化直接改了

File: README_CN.md [hunk 2]
```diff
@@ -167,8 +169,10 @@ configDataID: ServiceName.limit
 |failure_policy.backoff_policy| 可以设置的策略： `fixed` `none` `random` | 
 
 例子：
-```
-configDataId: ClientName.ServiceName.retry
+
+> configDataId: ClientName.ServiceName.retry
+
+```json
```
Comment (whalecold) [inline README_CN.md:175]: 这里的配置要更新吗

Comment (felix021) [reply README_CN.md:175]: 不需要，复用的是 StopPolicy.CBPolicy.ErrorRate 这个字段，原来是错误率熔断，换了个 retryContainer 的初始化方式以后，这个阈值用来限制 retry 比例了。

**Pull request merged**